camayoc/types/ui.py

```diff
@@ -125,10 +129,16 @@ def from_model(cls, model: Credential):
         except AttributeError:
             become_method = next(iter(NetworkCredentialBecomeMethods))
 
+        ssh_key = {
+            "ssh_key_file": model.ssh_keyfile,
+        }
+        if settings.camayoc.use_uiv2:
+            ssh_key["ssh_key_file_v2"] = server_container_ssh_key_content(model.ssh_keyfile)
+
         return cls(
             credential_name=model.name,
             username=model.username,
-            ssh_key_file=model.ssh_keyfile,
+            **ssh_key,
             passphrase=model.auth_token,
```

Comment:
oh, I thought ** could only be used after other kwargs. TIL

Reply:
Now that you mentioned it, I checked and that rule applies only to function signature / definition. Python will consider following to have syntax error:

```
def myfunc(**kwargs, d):
    pass
```

Also, while you can mix unpacking and keywords in function call, all arguments are still allowed only once. Following won't work:

```
def myfunc(a, b, c):
    pass

args = {"b": 1}
myfunc(a=1, b=2, **args, c=3)
```